### PR TITLE
Multi token

### PR DESCRIPTION
**Pull request by harture**

### CODE REVIEW

File: credentials.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+import (
+	"gopkg.in/h2non/gentleman.v2/plugins/body"
+	"gopkg.in/h2non/gentleman.v2/plugins/url"
+)
+
+const (
+	resetPasswordPath    = userIDPath + "/reset-password"
+	credentialsPath      = userIDPath + "/credentials"
+	credentialsTypesPath = realmPath + "/credentialTypes"
```
Comment (bsoniam) [inline credentials.go]: where is realmPath defined?

File: keycloak_client.go [hunk 2]
```diff
@@ -81,12 +80,22 @@ func New(config Config) (*Client, error) {
 		errTolerance = time.Minute
 	}
 
+	var httpClient = gentleman.New()
```
Comment (bsoniam) [inline keycloak_client.go]: Why did this part of code move? 
Just out of curiosity

File: keycloak_client.go [hunk 3]
```diff
@@ -32,9 +32,14 @@ type Config struct {
 type Client struct {
 	apiURL           *url.URL
 	httpClient       *gentleman.Client
+	account          *AccountClient
 	verifierProvider OidcVerifierProvider
 }
 
+type AccountClient struct {
```
Comment (bsoniam) [inline keycloak_client.go]: I am not used to this, but I don't have a better solution....for the moment....

File: keycloak_client.go (hunk 2 above)
Comment (harture) [reply keycloak_client.go]: It was just in order to keep the declaration of httpClient and its configuration close to each other

**Pull request merged**